wp-content/themes/ipbs-largo/less/mixins.less

```diff
@@ -19,9 +19,12 @@
     color: @white;
     font-weight: 600;
     border-radius: 20px!important;
-    padding: 10px 40px!important;
+    padding: 10px 50px!important;
     text-transform: uppercase;
     text-decoration: none;
+    box-shadow: 0px 0px 24px @box-shadow-black;
```

Comment:
![Screen Shot 2020-02-05 at 16 39 11](https://user-images.githubusercontent.com/1754187/73885650-2a19f700-4836-11ea-9e7a-76840309327e.png)

The `.btn` styles apply to the button in the mobile nav. 😢

Reply:
Should be fixed in https://github.com/INN/umbrella-ipbs/pull/83/commits/974bdf19a16285bf322d89cab2759077cd538b5f. Good catch 😅